serviceworker.js

```diff
@@ -11,38 +11,34 @@ importScripts('reconstructive.js');
 //   showBanner: false,
 //   debug: false
 // });
-let currentPath = self.location.href.substring(0, self.location.href.lastIndexOf('/'));
-Reconstructive.init({
+const currentPath = self.location.href.substring(0, self.location.href.lastIndexOf('/'));
+const reconstructive = new Reconstructive({
```

Comment:
Please change instance name to `rc` instead of `reconstructive` to avoid any confusion with the class name. Also, it will help when changing README documentation.

Reply:
done